app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -16,77 +18,103 @@ import {
   LabelsReferencesLastActive,
 } from "features/user/UserTextAndLabel";
 import { User } from "pb/api_pb";
-import { Link } from "react-router-dom";
-import { routeToUser } from "routes";
+import LinesEllipsis from "react-lines-ellipsis";
 import makeStyles from "utils/makeStyles";
 
 const useStyles = makeStyles((theme) => ({
-  card: {
-    borderRadius: theme.shape.borderRadius * 3,
+  root: {
+    textDecoration: "none",
+    // width: "100%",
```

Comment:
```suggestion
```
Doesn't look immediately obvious this will be needed even when we enable other community stuff back on the map?